### ISSUE TITLE
README.textile appears to be different from the implementation

### ISSUE THREAD

**Issue by pcn** [infochimps-labs/ironfan#52]
The README.textile indicates that cluster_chef is recording data in git.  From what I've found, role information is currently stored and queried in AWS server properties and merged with config settings (at least for knife, I assume the same is done for cluster_chef in general).

Specifically:

<pre>
* *Chef server is never the repository of truth* -- it only mirrors the truth.
  - a file is tangible and immediate to access
* Specifically, we want truth to live in the git repo, and be enforced by the chef server. *There is no truth but git, and chef is its messenger*.
  - this means that everything is versioned, documented and exchangeable.
* *Systems, services and significant modifications cluster should be obvious from the `clusters` file*.  I don't want to have to bounce around nine different files to find out which thing installed a redis:server.
  - basically, the existence of anything that opens a port should be obvious when I look at the cluster file.
</pre>

is this planned or is this approach discarded?  It would be very nice for cluster_chef to allow the state of a cluster at a point in time to be seen via git or some other dvcs since it would allow for the state of a cluster to be materialized and monitored from anywhere, not just via AWS tools, and without any aws authentication artifacts (keys, ids, etc.).

**Comment by mrflip** [comment]
By this we mean that all definitions _begin_ in the git repo -- everything in the chef server or AWS is either
- configuration, which came from something in the cluster definitions, cookbooks etc.
- intrinsic status reported by machines themselves

We store as little as possible in the AWS info -- there's some work required to remove the dependencies (#28) but there's nothing essential.
